.jenkins/branches.yaml.in

```diff
@@ -0,0 +1,10 @@
+upload-targets:
+  - name: local
+    method: rsync
+    dir: /h/binaries
+
+branches:
+  - name: hijinks.mit.edu/apt/autobuild
```

Comment:
I'm concerned about hard-coding the server that we're running on. Since you're already parameterizing this file, how hard would it be to also parameterize the server?